src/pydocstyle/config.py

```diff
@@ -23,6 +25,98 @@ def _decorator(self, *args, **kwargs):
     return _decorator
 
 
+class TomlParser:
+    """ConfigParser that partially mimics RawConfigParser but for toml files.
+
+    See RawConfigParser for more info. Also, please note that not all
+    RawConfigParser functionality is implemented, but only the subset, that is
```

Comment:
```suggestion
    RawConfigParser functionality is implemented, but only the subset that is
```
Small grammar nitpick coming through